<launch>
  <include file="$(find slaw_registration)/launch/icp.launch" />
  <include file="$(find slaw_navigation)/launch/slaw_global_navigation.launch" />
  <include file="$(find slaw_arm_navigation)/launch/slaw_arm_navigation.launch" />
  <!-- <include file="$(find slaw_object_detection)/launch/usb_cam.launch" /> -->

  <include file="$(find slaw_bringup)/launch/openni2.launch" />

  <node pkg="slaw_navigation" name="odom_move" type="odom_move_action.py" />
  <node pkg="slaw_navigation" name="align_line" type="align_line.py" />
  
  <node pkg="slaw_manipulation" name="detect_objects_depth" type="detect_objects_depth.py" respawn="true" output="screen" />
  <node pkg="slaw_manipulation" name="detect_objects_rgb" type="detect_objects_rgb.py" respawn="true" output="screen" />

  <node pkg="slaw_manipulation" name="detect_holes_rgb" type="detect_holes_rgb.py" respawn="true" output="screen" />
  <node pkg="slaw_manipulation" name="object_manager" type="object_manager.py" respawn="true" output="screen" />
  <node pkg="slaw_manipulation" name="hole_manager" type="holes_manager.py" respawn="true" output="screen" />
  
  <rosparam command="load" file="$(find slaw_smach)/config/test.yaml" />
  <rosparam command="load" file="$(find slaw_smach)/config/holes.yaml"/>
</launch>
